examples/basic/package.json

```diff
@@ -10,7 +10,7 @@
     "vite": "^2.7.2"
   },
   "dependencies": {
-    "@uhlive/javascript-sdk": "^0.18.0",
-    "keycloak-js": "^17.0.1"
+    "@uhlive/javascript-sdk": "^1.0.0-rc1",
```

Comment:
If you want to use the `1.0.0-rc1` version, we shouldn't merge in `main`

Reply:
I was completely lost reading your comment... until I realized I messed up somewhere and that the SSO support has been merged in the `main` branch... we didn't realized it because the SDK support both auth methods...